tests/environment-install.bats

```diff
@@ -74,10 +74,10 @@ teardown() {
 
 @test "'flox uninstall' edits manifest" {
   "$FLOX_CLI" init;
-  run "$FLOX_CLI" install foo;
+  run "$FLOX_CLI" install hello;
   assert_success;
-  run "$FLOX_CLI" uninstall foo;
-  run grep "foo = {}" "$PROJECT_DIR/.flox/env/manifest.toml";
+  run "$FLOX_CLI" uninstall hello;
+  run grep "^hello = {}" "$PROJECT_DIR/.flox/env/manifest.toml";
```

Comment:
suggestion blocking: add a test

Reply:
I'm confused, we already have tests for this. I made the change here specifically because the test fails with package `foo`, which doesn't exist.

Reply:
The existing tests only check that the manifest gets modified. You could check:
1. Installing an invalid package fails with a helpful message
2. `$PROJECT_DIR/.flox/run/$system.$name/bin` contains `hello`